src/js/editor/editor.js

```diff
@@ -754,8 +754,7 @@ class Editor {
     let pastedPost = parsePostFromPaste(event, this.builder, this._parserPlugins);
 
     this.run(postEditor => {
-      let nextPosition = postEditor.insertPost(position, pastedPost);
-      postEditor.setRange(new Range(nextPosition));
```

Comment:
Can this be left outside `insertPost`?

We discussed the range stuff in detail but I don't remember if we talked about moving cursor changes into `postEditor` methods. Am I forgetting a reason this must be done?